src/components/scene/ar-hit-test.js

```diff
@@ -37,6 +61,36 @@ HitTest.prototype.sessionStart = function sessionStart (options) {
   }
 };
 
+HitTest.prototype.anchorFromLastHitTestResult = function (options) {
```

Comment:
We can perhaps make the parameters explicit instead of passing an object. It makes the function more legible / easier to use?

Reply:
Sure, this is a hangover from when I had destructuring before.